src/Illuminate/Routing/Middleware/ValidateSignature.php

```diff
@@ -12,13 +12,15 @@ class ValidateSignature
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Closure  $next
+     * @param  \string|\null $relative Should be the word "relative" or null.
      * @return \Illuminate\Http\Response
      *
      * @throws \Illuminate\Routing\Exceptions\InvalidSignatureException
      */
-    public function handle($request, Closure $next)
+    public function handle($request, Closure $next, $relative = null)
```

Comment:
This breaks compatibility with L6. It would be better to use `$relative = func_get_arg(3);` if the middleware handling supports it.

Reply:
Heya. This would indeed need to go to master because it breaks the message signature.

Reply:
so I should change the base to master and change aim for L7?

Reply:
Either that or make it compatible with L6, as I'm pretty sure @driesvints meant the method sinature.

Have a look at [accessing function parameters without changing signature](https://www.php.net/manual/en/function.func-get-arg.php).

If you chose the latter don't forget to send another pr to change the signature when this was merged.